src/templates/admin/elements/removable_role_chips.html

```diff
@@ -0,0 +1,28 @@
+<form method="POST">
+  <div class="flex">
+    {% csrf_token %}
+    <input hidden name="params_string" value="{{ params_string }}">
+    {% for accountrole in accountroles %}
+      {% if accountrole.journal == journal %}
+        <div class="small button-group">
+          <span class="small button secondary chip">
+            {{ accountrole.role.name }}
+          </span>
+          <button
```

Comment:
Interesting design choice. I find the <span> elements styled as buttons a bit confusing, do we really need the split instead of having a single button? It reminds me of the accesibility issues that often arise around [checkboxes and their labels](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/checkbox_role)